arangod/RestServer/DatabaseFeature.cpp

```diff
@@ -309,7 +309,7 @@ void DatabaseFeature::collectOptions(
       .setLongDescription(R"(If the maximum number of databases is reached, no
 additional databases can be created in the deployment. In order to create additional
 databases, other databases need to be removed first.")")
-      .setIntroducedIn(31120);
+      .setIntroducedIn(31102);
```

Comment:
NIT it's more readable and less error prone
```suggestion
      .setIntroducedIn(3'11'02);
```

Reply:
That would require more general change but could be considered in a separate devel PR.